consensus/ibft/sign.go

```diff
@@ -110,8 +110,7 @@ func writeCommittedSeals(h *types.Header, seals [][]byte) (*types.Header, error)
 	return h, nil
 }
 
-// TODO: This function is called signHash, we are doing no signing
-func signHash(h *types.Header) ([]byte, error) {
+func getHeaderHash(h *types.Header) ([]byte, error) {
```

Comment:
What do you think if we rename this to `calculateHeaderHash()`?
As it is now, I would expect it to only do `return h.Hash`

Reply:
Agree with Vuksan on this one, `get` should only be reserved for getter methods and simple helper funcs.